.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,14 +3,14 @@ CBG-0000
 Describe your PR here...
 - Use bullet points if there's more than one thing changed
 
-## Pre-review checklist (remove once done)
```

Comment:
I think we should leave it here but just remove the `(remove once done)` part, and make each item a checkbox.

It really should be the developers responsibility to make sure this is done before it's considered ready for review, but can't harm to leave the list to remind the reviewer to check.